beets/library.py

```diff
@@ -747,6 +747,20 @@ def move_file(self, dest, operation=MoveOperation.MOVE):
             util.hardlink(self.path, dest)
             plugins.send("item_hardlinked", item=self, source=self.path,
                          destination=dest)
+        elif operation == MoveOperation.REFLINK:
+            util.reflink(self.path, dest, fallback=False)
+            plugins.send("item_reflinked", item=self, source=self.path,
+                         destination=dest)
+        elif operation == MoveOperation.REFLINK_AUTO:
+            util.reflink(self.path, dest, fallback=True)
+            plugins.send("item_reflinked", item=self, source=self.path,
+                         destination=dest)
+        else:
```

Comment:
Do you remember the reason the move stuff needed to be added? (this and line 1108)

I'm not too familiar with this module so I'm wondering how/if it pertains to the reflink logic.

Reply:
I have no clue any more! Maybe it's an rebase artefact of a previous attempt, where `move` was the fallback in those methods? If you, nor @sampsyo see why this is there, maybe it should just go.

On the other hand, are we sure that the `operation` variable is checked exhaustively?

Reply:
Interesting! I'm similarly stumped. I say we replace this with:

```py
else:
    assert False, 'unknown MoveOperation'
```

…to check exhaustiveness.